GITEA_GUIDE.md

```
@@ -0,0 +1,220 @@
+4. **Run Playbooks with Decryption Key**:
+   ```bash
+   ansible-playbook ansible/setup_gitea.yml --ask-vault-pass
+   # Or using a secure local password file (excluded from Git):
+   ansible-playbook ansible/setup_gitea.yml --vault-password-file ~/.gitea_vault_pass.txt
+   ```
+
+### Method 2: Runtime Environment Variables (Dynamic Ingestion)
+Instead of committing passwords, inject them dynamically from the active runtime environment using the Ansible environment lookup plugin:
+
+1. **Configure Variable Lookup in the Playbook**:
+   ```yaml
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)